src/hpc-cache/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1

```

Comment:
Leaving it would make the wheel name built out like `hpc-cache-0.1.0-py2-py3-none-any.whl`.

However azure-cli-core has dropped Python 2 supprt, we should leave this file empty.